StructureDefinition/England-MedicationRequest.xml

```diff
@@ -0,0 +1,649 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="England-MedicationRequest" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/England-MedicationRequest" />
+  <version value="1.0.0" />
+  <name value="EnglandMedicationRequest" />
+  <title value="England MedicationRequest" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="This profile defines the NHS England constraints and extensions on the FHIR resource." />
```

Comment:
Description massively changed? Should be like the previous version where it has a link to the Simplifier page.